src/control/cmd/daos_server/storage.go

```diff
@@ -120,22 +118,56 @@ func (cmd *storagePrepScmCmd) Execute(args []string) (err error) {
 		return errors.WithMessage(err, "initialising ControlService")
 	}
 
-	needsReboot, devices, err := svc.PrepareScm(server.PrepareScmRequest{
-		Reset: cmd.ResetScm,
-	})
-	if err != nil {
-		return err
-	}
+	cmd.log.Info("Preparing locally-attached storage...")
+
+	scanErrors := make([]error, 0, 2)
+	e := make(chan error)
```

Comment:
This is an unbuffered channel, which means that these goroutines will block until the parent function reads the channel. Not sure if this is what you intend.

Reply:
that is what I intended, is there problem with that?